### DISCUSSION TITLE
ION (WPXQ) Providence

### DISCUSSION THREAD

**Discussion by spotanjo3** [iptv-org/community#767] [Q&A]
### Channel Name

ION (WPXQ) Providence

### Channel ID

WPXQDT1.us

### Website

https://www.iontelevision.com

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by spotanjo3** [comment]
I am anxiously and waiting for it. :D

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19760
